test/container_app_test.py

```diff
@@ -71,6 +72,20 @@ async def stop_app(client, app_id):
     return await retry_transient_errors(client.stub.AppStop, api_pb2.AppStopRequest(app_id=app_id))
 
 
+@contextmanager
+def set_env_vars(restore_path, container_addr):
+    with mock.patch.dict(
+        os.environ,
+        {
+            "MODAL_RESTORE_STATE_PATH": str(restore_path),
+            "MODAL_SERVER_URL": container_addr,
```

Comment:
Are we aiming to mock the container environment here? We don't supply this environment variable to the container AFAIK.